CMSIS/NN/Source/SoftmaxFunctions/arm_softmax_s8.c

```diff
@@ -28,10 +28,50 @@
  *
  * -------------------------------------------------------------------- */
 
-#include "arm_nnfunctions.h"
+#include "arm_nnsupportfunctions.h"
```

Comment:
Why not arm_nnfunctions?

Reply:
arm_nnfunctions.h is the external API interface to CMSIS-NN which also includes arm_nnsupportfunctions.h. Including just arm_nnsupportfunctions.h is a way of saying that this is a standalone function that does not depend on any of the functions in the external API list.